crates/tabby/src/services/completion/completion_prompt.rs

```diff
@@ -182,7 +182,7 @@ async fn collect_snippets(
     content: &str,
 ) -> Vec<Snippet> {
     let query = CodeSearchQuery {
-        git_url: git_url.to_owned(),
+        git_url: RepositoryConfig::canonicalize_url(git_url),
```

Comment:
CodeSearch already handles canonicalization / closest_match internally, no need to make this change.

https://demo.tabbyml.com/files/github/TabbyML/tabby/-/blob/48d9c084cc80d20bd439102cc0768920f6680e91/crates/tabby/src/services/code.rs?plain=1#L78